app/controllers/providers/proceedings_sca/confirm_delete_core_proceedings_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Providers
+  module ProceedingsSCA
+    class ConfirmDeleteCoreProceedingsController < ProviderBaseController
+      prefix_step_with :proceedings_sca
+
+      def show
+        proceeding
+      end
+
+      def destroy
+        remove_proceedings
+        replace_last_page_in_history(providers_legal_aid_applications_path)
```

Comment:
Discussion: This is the pattern for how we handle the back link when removing something in other places in the application e.g. in remove_dependants_controller, remove_state_benefits_controller and remove_vehicles_controller. It takes the user back to their list of applications (in this case, if they click the back link after removing a core proceeding) and then they can't click back again. I think this might be the best we can do for this use case as they could have come to the confirm_delete_core_proceedings page via various routes e.g. by creating related proceedings which we are also deleting so the page history would also be invalid for these pages. I think trying to untangle that logic would get messy?

Reply:
Seems sensible to me :+1: